spec/javascripts/highlight-active-section-heading-spec.js

```diff
@@ -0,0 +1,111 @@
+            <p>Without it, you won’t know what to build, how to build it or what problems you’re trying to solve.</p>\
+            <h2 id="meeting-the-digital-service-standard">Meeting the Digital Service Standard</h2>\
+            <p>To pass <a href="/service-manual/digital-by-default#criterion-1">point 1 (understand user needs)</a> in your <a href="/service-manual/service-assessments/check-if-you-need-a-service-assessment">service assessments</a>, you must show that you’ve researched the needs of all your users, including those with the lowest level of digital access, skill and literacy.</p>\
+            <p>To pass <a href="/service-manual/digital-by-default#criterion-2">point 2 (do ongoing research)</a> you must show that you have an ongoing plan to research and test your service with all your users, including those with access and support needs.</p>\
+            <h2 id="understand-your-users">Understand your users</h2>\
+            <p>To deliver a service that meets your <a href="/service-manual/user-research/start-by-learning-user-needs">users’ needs</a>, you have to understand:</p>\
+            <ul>\
+              <li>who your likely users are</li>\
+              <li>what they’re trying to do</li>\
+              <li>how they’re trying to do it now</li>\
+            </ul>\
+            <p>Unlike market research, you need to work out how your users think and behave rather than find out what they like or think they want.</p>\
+            <p>If you understand your users’ circumstances, influences and expectations, it will help you design a service that’s easy to use and valued by the people who need it.</p>\
+          </div>\
+        </div>\
+      </div>\
+    </div>')
+
+    describe('On desktop', function () {
+      beforeEach(function () {
+        module.getWindowDimensions = function () {
+          return {
+            height: 768,
+            width: 1024
+          }
+        }
+        module.getHeadingPosition = function (theID) {
+          return {
+            top: 200
+          }
+        }
+        module.getNextHeadingPosition = function (theNextID) {
+          return {
+            top: 400
+          }
+        }
+      })
+
+      it('has no highlighted nav items, when the page loads', function () {
+        module.getWindowPositions = function () {
+          return {
+            scrollTop: 0
+          }
+        }
+        module.start($element)
+        var $anchors = $element.find('.js-page-contents a')
+        expect($anchors.hasClass('active')).toBe(false)
+      })
+
+      it('highlights a nav item, when the page is scrolled', function () {
+        module.getWindowPositions = function () {
+          return {
+            scrollTop: 300
+          }
+        }
+        module.start($element)
+        var $anchor = $element.find('.js-page-contents a')
```

Comment:
It'd be good if we were more explicit about which link has the 'active' class, rather than assert that one of the links is active.